apps/wing-console/console/ui/src/features/map-view.tsx

```diff
@@ -98,9 +108,33 @@ const Wrapper: FunctionComponent<PropsWithChildren<WrapperProps>> = memo(
           >
             {name}
           </span>
+          <div className="flex grow justify-end">
+            <div
+              className="pl-1"
+              onClick={() => {
+                onCollapse(!collapsed);
+              }}
+            >
+              {collapsed && (
+                <ChevronDownIcon
+                  className={clsx(
+                    "size-4",
+                    "hover:text-sky-600 dark:hover:text-sky-300 transition-colors",
+                  )}
+                />
+              )}
+              {!collapsed && (
+                <ChevronUpIcon
+                  className={clsx(
+                    "size-4",
+                    "hover:text-sky-600 dark:hover:text-sky-300 transition-colors",
+                  )}
+                />
+              )}
+            </div>
+          </div>
         </div>
-
-        {children}
+        {collapsed || children}
```

Comment:
Won't this render something like a `1` or `true` if collapsed? 🤔

Reply:
React is not rendering booleans 
<img width="872" alt="image" src="https://github.com/winglang/wing/assets/5547636/a79e0248-dc87-4628-851d-b8919cd27f15">

Reply:
Perfect